coverport/coverage_server.py

```
@@ -0,0 +1,54 @@
+                    import tempfile
+
+                    with tempfile.NamedTemporaryFile(mode="w", suffix=".lcov", delete=False) as f:
+                        cov.lcov_report(outfile=f.name)
+                        data = open(f.name).read()
+                        os.unlink(f.name)
+                    self.send_response(200)
+                    self.send_header("Content-Type", "text/plain")
+                    self.end_headers()
+                    self.wfile.write(data.encode())
+                    return
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)